types/connect-flash-plus/index.d.ts

```diff
@@ -0,0 +1,24 @@
+// Type definitions for connect-flash-plus 0.2
+// Project: https://github.com/martinkadlec0/connect-flash-plus
+// Definitions by: Michael V. Mykhaylov <https://github.com/MikeMikhailov>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.7
+
+/// <reference types="express" />
+
+declare namespace Express {
+    interface Request {
+        flash(): { [key: string]: string[] };
+        flash(type?: string, msg?: string | string[], ...args: string[]): any;
+    }
+}
+
+// tslint:disable-next-line:no-declare-current-package no-single-declare-module
```

Comment:
instead of this, I think you should be able to wrap `declare namespace Express` inside a `declare global {` and then put all the contents of the below block at the top-level. Does that work?